authors/settings.py

```diff
@@ -158,3 +158,9 @@
     #     'authors.apps.authentication.backends.JWTAuthentication',
     # ),
 }
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = config('EMAIL_PORT')
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = config('EMAIL_HOST_USER')
+EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
```

Comment:
@moseskamira Could you create an email for the guardians that will be used for this. e.g gauthorshaven@gmail.com

Reply:
created and PR Information editted accordingly